### DISCUSSION TITLE
First item of second column underlined

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [sxyazi/yazi#485] [1. Q&A] [answered]
![image](https://github.com/sxyazi/yazi/assets/61395246/b594b092-2e6b-4a7a-8455-177270411d6c)
Why is the first item of the second column always underlined and white?

**Comment by lpnh** [comment]
I had never noticed it. Wish I could unsee this. My life was happier before.

**Comment by sxyazi** [comment, marked as answer]
This is intentional; I wanted to add an underline instead of a background color to indicate it's a "temporary view" during preview.

Feel free to change its style to your liking easily:

```bash
# ~/.config/yazi/theme.toml
[manager]
preview_hovered = { fg = "black", bg = "lightblue" }
```

Just add the above to your configuration file. See the docs here: https://yazi-rs.github.io/docs/configuration/theme#manager

**Reply by LamprosPitsillos** [reply]
Thank you so much, Yazi is incredible 🚀
